scripts/migrate.js

```diff
@@ -19,6 +19,7 @@ const q = faunadb.query;
     NEXT_PUBLIC_FAUNA_GRAPHQL_DOMAIN,
     FAUNA_DOMAIN,
     FAUNA_SECRET_ADMIN,
+    OVERRIDE_SCHEMA
```

Comment:
The override flag should not be an environment variable. It should be a flag passed on invocattion.